_includes/head.html

```diff
@@ -28,6 +28,9 @@
     })(Tc.$);
   </script>
   <link href="http://fonts.googleapis.com/css?family=Raleway:400,700,300" media="screen" rel="stylesheet" type="text/css" />
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/axios/0.19.2/axios.min.js"></script>
```

Comment:
if you get axios from cloudflare, you shouldn't have to include it (next file).

Reply:
what do you mean not include it? just put it in the index.html instead of head.html?